tests/framework/vddk.go

```diff
@@ -12,6 +12,38 @@ import (
 	"kubevirt.io/containerized-data-importer/tests/utils"
 )
 
+// CreateVddkDataVolume returns a VDDK data volume
+func (f *Framework) CreateVddkDataVolume(dataVolumeName, size, url string) *cdiv1.DataVolume {
```

Comment:
How come something like this didn't already exist?

Reply:
It does, but only in datavolume_test. I moved this test to import_test for some reason, I guess because I couldn't shoehorn the log scanning nicely into the big testDataVolume table. I moved it back to get rid of this change, it seems like it doesn't really belong in import_test any more than datavolume_test.